package.json

```diff
@@ -2,13 +2,16 @@
   "name": "backpat",
   "version": "0.5.0",
   "description": "A simple tool for high-fiving your tech stack",
-  "main": "src/index.js",
+  "main": "dist/index.js",
   "scripts": {
-    "test": "nyc mocha",
+    "prebuild": "rimraf dist",
+    "build": "npm-run-all --parallel build:*",
+    "build:main": "babel --out-dir dist --ignore *.test.js src",
+    "test": "cross-env NODE_ENV=test nyc mocha",
```

Comment:
Sets the node environment variable to test. Because this process is different across platforms, a library like cross-env is necessary to detect the developers platform for testing. If the environment variable is not set to test, our instrumented code would be outputted to dist.